pmg/views.py

```diff
@@ -345,6 +346,18 @@ def committee_detail(committee_id):
     )
 
 
+@app.route('/committee/<int:committee_id>/follow-cte')
+@app.route('/committee/<int:committee_id>/follow-cte/')
+def committee_detail_follow_cte(committee_id):
+    follow_committee(committee_id)
+    
+    # Remove confirmation email flash
+    session['_flashes'] = []
```

Comment:
Don't we want to show this?

Reply:
We're already showing a message about it at the top of the page. I just thought it looks rather busy with all the notifications.

![screenshot at jun 21 09-36-21](https://user-images.githubusercontent.com/11753432/41704498-ab236de2-7536-11e8-83da-4efe631b023f.png)